x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ilm/TimeseriesLifecycleTypeTests.java

```diff
@@ -492,7 +493,7 @@ private void assertInvalidAction(String phaseName, String currentAction, String.
             case DeleteAction.NAME:
                 return new DeleteAction();
             case ForceMergeAction.NAME:
-                return new ForceMergeAction(1);
+                return new ForceMergeAction(1, Codec.getDefault());
```

Comment:
These should pass null (or have a second constructor so there's no need to specify the codec)